rootfs/regular/runtimes/python3/workload.py

```diff
@@ -5,6 +5,13 @@
 import json
 import time
 import socket
+import os
+from subprocess import run, Popen
+
+# send over the boot completion signal
+for i in range(1, os.cpu_count()):
+    Popen('taskset -c {} outl 123 0x3f0'.format(i), shell=True)
+run('taskset -c 0 outl 123 0x3f0', shell=True)
```

Comment:
The signal should be after `import_module`

Reply:
Resolved. New results coming up. For the results so far, It seems that `import_module` is quite heavy. Around 300ms for boot, 5ms for vsock connection, and the rest goes to import_module. Especially in the sentiment-analysis case, it's 1000ms.

Reply:
it is application specific since it is loading function logic.